recipes/gvisor/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set provider = "gvisor" %}
```

Comment:
The provider variable makes more sense for terraform modules. It is not necessary in this case.
```suggestion
{% set name = "gvisor" %}
```